tests/bsim/bluetooth/ll/cis/src/main.c

```diff
@@ -49,22 +52,22 @@ static bt_addr_le_t peer_addr;
 #define CREATE_CONN_WINDOW   0x0010
 
 #define ISO_INTERVAL_US      10000U
+#define ISO_INTERVAL_MS      (ISO_INTERVAL_US / USEC_PER_MSEC)
 #define ISO_LATENCY_MS       DIV_ROUND_UP(ISO_INTERVAL_US, USEC_PER_MSEC)
 #define ISO_LATENCY_FT_MS    20U
 
-#define BT_CONN_US_TO_INTERVAL(t) ((uint16_t)((t) * 4U / 5U / USEC_PER_MSEC))
-
 #if (CONFIG_BT_CTLR_CENTRAL_SPACING == 0)
-#define CONN_INTERVAL_MIN    BT_CONN_US_TO_INTERVAL(ISO_INTERVAL_US)
+#define CONN_INTERVAL_MIN_MS ISO_INTERVAL_MS
 #else /* CONFIG_BT_CTLR_CENTRAL_SPACING > 0 */
-#define CONN_INTERVAL_MIN    BT_CONN_US_TO_INTERVAL(ISO_INTERVAL_US * CONFIG_BT_MAX_CONN)
+#define CONN_INTERVAL_MIN_MS (ISO_INTERVAL_MS * CONFIG_BT_MAX_CONN)
 #endif /* CONFIG_BT_CTLR_CENTRAL_SPACING > 0 */
 
+#define CONN_INTERVAL_MIN    BT_GAP_MS_TO_CONN_INT(CONN_INTERVAL_MIN_MS)
 #define CONN_INTERVAL_MAX    CONN_INTERVAL_MIN
-#define CONN_TIMEOUT         MAX((BT_CONN_INTERVAL_TO_MS(CONN_INTERVAL_MAX) * 6U / 10U), 10U)
+#define CONN_TIMEOUT         BT_GAP_MS_TO_CONN_TIMEOUT(MAX((CONN_INTERVAL_MIN_MS * 6U), 100U))
```

Comment:
Since `CONN_INTERVAL_MIN` and `CONN_INTERVAL_MAX` are the same it doesn't make a difference in practice, but shouldn't this really be (where CONN_INTERVAL_MAX_MS needs to be defined yet):

```suggestion
#define CONN_TIMEOUT         BT_GAP_MS_TO_CONN_TIMEOUT(MAX((CONN_INTERVAL_MAX_MS * 6U), 100U))
```